### ISSUE TITLE
Generate RTD documentation from inline documentation

### ISSUE THREAD

**Issue by WalterKolczynski-NOAA** [NOAA-EMC/global-workflow#1380]
**Description**
HTML documentation needs to be generated from the planned inline reST documentation.

**Requirements**
Create script(s) to generate HTML documentation suitable for ReadTheDocs using Sphinx or another documentation builder like Doxygen.

**Acceptance Criteria (Definition of Done)**
ReadTheDocs documentation generated from inline documentation of source code.

**Dependencies**
Issue #1327

**Comment by HenryRWinterbottom** [comment]
@WalterKolczynski-NOAA What scripts did you want to generate in-line docs for? Those beneath `jobs/`, `scripts/`, and/or something else?

**Comment by WalterKolczynski-NOAA** [comment]
Yeah, those in `job`, `scripts`, and `ush`. Might be hard before #1327, but maybe if you concentrate on the python ones for now, which we generally made sure came in documented, unlike the bash which we need to go back and do.
